fcos-graph-builder/src/main.rs

```diff
@@ -132,9 +146,26 @@ pub(crate) async fn gb_serve_graph(
     Ok(resp)
 }
 
+/// CLI configuration options.
 #[derive(Debug, StructOpt)]
 pub(crate) struct CliOptions {
+    /// Verbosity level (higher is more verbose).
+    #[structopt(short = "v", parse(from_occurrences))]
+    verbosity: u8,
+
     /// Path to configuration file.
```

Comment:
Just realized https://github.com/openshift/cincinnati/blob/master/dist/openshift/cincinnati.yaml#L236 for Cincinnati config setup. Really interesting :)

Reply:
Yes, that's the target. For policy-engine it should be easy to share the same config mostly, for graph-builder I'd need to check how to fit the settings for multiple streams into a single file.